microprofile-fault-tolerance/README.adoc

```diff
@@ -622,28 +622,18 @@ See details in the section <<adding_resiliency_circuit_breakers, Adding Resilien
 // Undeploy the Quickstart
 include::../shared-doc/undeploy-the-quickstart.adoc[leveloffset=+2]
 
-// Run the Arquillian Tests
-include::../shared-doc/run-arquillian-tests.adoc[leveloffset=+2]
+// Server Distribution Testing
+include::../shared-doc/run-integration-tests-with-server-distribution.adoc[leveloffset=+2]
 
 // Run the Quickstart in Red Hat CodeReady Studio or Eclipse
```

Comment:
since you are going to do changes, please remove the include of the deprecated content.

Reply:
I removed the "Run the Quickstart in Red Hat CodeReady Studio or Eclipse" inclusion. I hope that's what you meant?